src/styles/components/dialog-screen.styl

```diff
@@ -2,7 +2,7 @@
   position: fixed
   left: 0
   top: 0
-  z-index: 1000000  //Place waaay above map plugin elements.
+  z-index: 3000000  //Place waaay above map plugin elements.
```

Comment:
@shaunanoordin (not a blocker) Why is such a big number needed? We don't use `z-index` anywhere else, do we? If a high number is needed, we could follow some kind of convention like https://css-tricks.com/rational-z-index-values/ or https://css-tricks.com/handling-z-index/